src/components/dashboard/dashboard-list/DashboardList.tsx

```diff
@@ -38,21 +44,269 @@ class DashboardList extends Component<DashboardListProps, DashboardListState> {
+                        applicationScore={this.props.reviewed[index].applicationScore}
+                        interviewScore={this.props.reviewed[index].interviewScore}
+                        viewApplicant={this.props.viewApplicant}
+                        setModalAndType={(type: string) => {
+                            console.log(element.role);
+                            this.showModal(element.name, element.role, type);
+                        }}
+                    />
+                ));
+            case 'Scheduled For Interview':
+                return this.props.scheduled.map((element, index) => (
+                    <DashboardListCard
+                        {...element}
+                        mode={this.props.mode}
+                        key={index}
+                        count={index}
+                        applicationScore={this.props.scheduled[index].applicationScore}
+                        interviewScore={this.props.scheduled[index].interviewScore}
+                        viewApplicant={this.props.viewApplicant}
+                        setModalAndType={(type: string) => {
+                            console.log(element.role);
+                            this.showModal(element.name, element.role, type);
+                        }}
+                    />
+                ));
+            case 'Interviewed':
+                return this.props.interviewed.map((element, index) => (
+                    <DashboardListCard
+                        {...element}
+                        mode={this.props.mode}
+                        key={index}
+                        count={index}
+                        applicationScore={this.props.interviewed[index].applicationScore}
+                        interviewScore={this.props.interviewed[index].interviewScore}
+                        viewApplicant={this.props.viewApplicant}
+                        setModalAndType={(type: string) => {
+                            console.log(element.role);
+                            this.showModal(element.name, element.role, type);
+                        }}
+                    />
+                ));
+            case 'Final Decision':
+                return this.props.accepted.map((element, index) => (
+                    <DashboardListCard
+                        {...element}
+                        mode={this.props.mode}
+                        key={index}
+                        count={index}
+                        applicationScore={this.props.accepted[index].applicationScore}
+                        interviewScore={this.props.accepted[index].interviewScore}
+                        viewApplicant={this.props.viewApplicant}
+                        setModalAndType={(type: string) => {
+                            console.log(element.role);
+                            this.showModal(element.name, element.role, type);
+                        }}
+                    />
+                ));
+        }
+    };
```

Comment:
Same as above with this conditional render

Reply:
Should I just move this logic into DashboardListCard or create a brand new component?

Reply:
I don't think I looked at this block super closely, this is fine. Adding a new comment to describe what I mean with the previous block.